example/spec/loadTest/ingestLoad_queueGranPassThruSF.js

```diff
@@ -0,0 +1,253 @@
+const fs = require('fs');
```

Comment:
let's think about the naming here. In 6 months I'm not sure we're going to know what `ingestLoad_queueGranPassThruSF` means. What about something like `queueGranulePassThroughLoadTest`? Same would apply to the other tests.

Reply:
The "Pass Through" concept is also potentially confusing if you don't have all of the context so I'm open to other ideas.

Reply:
I see your good point. Im open to the name change. Note that the queueGranPassThruSF = QueueGranulesPassthrough Step Functions which is the actual existing workflow name in Core currently, however. 
Same as the queueGranSF being the name of the existing QueueGranules workflow. So the naming kinda indicates the usage of existing workflows. But suggestions are welcome.

Reply:
Changed script names as suggested.